style.css

```diff
@@ -367,3 +370,137 @@ textarea {
   font-weight: 700;
   color: #172b4d;
 }
+
+/* RESPONSIVE */
+
+.only-desktop {
+  display: none;
+}
+
+@media only screen and (min-device-width: 780px) {
```

Comment:
- [ ] As stated by the previous reviewer, the media query should lie between `992px `and `1100px.` Please ensure you achieve that.